jms/src/main/scala/akka/stream/alpakka/jms/impl/Sessions.scala

```diff
@@ -99,6 +99,8 @@ private[jms] final class JmsAckSession(override val connection: jms.Connection,
     try {
       if (session.getAcknowledgeMode() == jms.Session.CLIENT_ACKNOWLEDGE)
         drainAcks()
+    } catch {
+      case ex: JMSException if ex.getMessage.contains("JMSCC0033") => // Swallow as ack isn't necessary if this is being thrown
```

Comment:
Can we be sure `getMessage` isn't `null`?

Reply:
`JMSException` does not have an empty constructor, but there is no check that the provided value is not `null`. I do not believe it is used in the capacity where `null` is passed in, but I added the check to be safe.